card/src/main/java/com/adyen/checkout/card/internal/ui/view/CardNumberOutputTransformation.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 13/10/2025.
+ */
+
+package com.adyen.checkout.card.internal.ui.view
+
+import androidx.compose.foundation.text.input.OutputTransformation
+import androidx.compose.foundation.text.input.TextFieldBuffer
+import androidx.compose.foundation.text.input.insert
+
+internal class CardNumberOutputTransformation(
```

Comment:
They say it's better for the compose performance if these `OutputTransformation`s are `data class`es and are marked with `@Stable`